### ISSUE TITLE
Default category can't be removed

### ISSUE THREAD

**Issue by EdelReid** [LNReader/lnreader#611]
I have sorted out my novels among 4 categories in the library. But there's a 5th category, which I have no use of, 'default' category. It can't be deleted, nor can I change its name to replace it with others. It stays there along with other categories, and this irks me a lot. Even in the library section in the settings, it says the number of categories are 5, but only 4 can be shown, that can be edited or sorted in an order. 

### Expected behavior

It would be helpful if we have the option to remove the default category. 

### Actual behavior

-

### LNReader version

1.1.15

### Android version

Android 10

### Device

Honor 8x

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with a source, I should be opening an issue in the [sources repository](https://github.com/lnreader/lnreader-sources/issues/new/choose).
- [X] I have updated the app to version **[1.1.15](https://github.com/lnreader/lnreader/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by EdelReid** [comment]
Hey, I can see that the default category has been removed. Thanks for that. 

There thing is now i can't see any novel entries in my library. It says, "Your library is empty. Add series to your library from browse." However, somehow I can update (refresh) those invisible entries. In other words, those entries are there in the library, they are updated on daily basis, can be seen in the updates section, but it's not showing on the library screen. I tried backing up from the previous saved file, and still not working.

**Comment by CD-Z** [comment]
@EdelReid The default category can't be removed, however it won't be displayed if there are no novels inside. If you select another category as default category it also won't appear again when you add a new novel.

**Comment by EdelReid** [comment]
Thanks got the reply. I got the result, the default category can't be
viewed from the library now.

I remade one of the categories which I believed the 'default' category,
literally was showing the same Novels. So now I made another category to be
default, and now I can't see the literal 'default' category.

In doing so, and even now, I encountered another issue. Whenever I try to
press and hold the 'add to library' in the 'global search' and tick mark
one of the categories, the novel doesn't appear in the library in that
category. So overall now press and hold doesn't work as intended in the
global search. Simply clicking 'add to library' adds the novel to the
category which I have set to default. So once the novel has been added to
the default category, after going to that particular novel in the library,
i can then press and hold and transfer it between categories. Hope this
helps.
